app-crypt/moolticute/moolticute-0.42.1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
I think 7 works here. You can use `xdg-utils.eclass` instead of `gnome2-utils` to call for `xdg_icon_cache_update`. The icon_cache_update has migrated away from gnome2-utils.

Rest looks good to me!

Reply:
Updated! Indeed, it works as well :)